### ISSUE TITLE
Eve 0.7.9 and Flask-PyMongo 2.0.0 are not compatible

### ISSUE THREAD

**Issue by mohierf** [pyeve/eve#1172]
My application broke as soon as Flask-PyMongo 2..0.0 got released (as of #1166). I am currently using Eve 0.7.9 and I am stucked to this version.

Indeed Eve do not set any version requirement on PyMongo except for > 0.4. Thus, when PyMongo released the 2.0.0 version, this new release is installed with the upgrade and the Eve backend do not start any more.

The error raised is:
```
MongoDB connection string: mongodb://mongoquery1:27017,mongoquery2:27017
Traceback (most recent call last):
  File "/usr/local/etc/alignak-backend/alignak-backend.wsgi", line 4, in <module>
    from alignak_backend.app import app as application
  File "/usr/local/lib/python2.7/site-packages/alignak_backend/app.py", line 2158, in <module>
    static_folder=base_path
  File "/usr/local/lib/python2.7/site-packages/eve/flaskapp.py", line 176, in __init__
    self.register_resource(resource, settings)
  File "/usr/local/lib/python2.7/site-packages/eve/flaskapp.py", line 922, in register_resource
    create_index(self, resource, name, list_of_keys, index_options)
  File "/usr/local/lib/python2.7/site-packages/eve/io/mongo/mongo.py", line 969, in create_index
    db = app.data.pymongo(resource, px).db
  File "/usr/local/lib/python2.7/site-packages/eve/io/mongo/mongo.py", line 893, in pymongo
    self.driver[px] = PyMongo(self.app, px)
  File "/usr/local/lib/python2.7/site-packages/flask_pymongo/__init__.py", line 116, in __init__
    self.init_app(app, uri, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/flask_pymongo/__init__.py", line 149, in init_app
    parsed_uri = uri_parser.parse_uri(uri)
  File "/usr/local/lib/python2.7/site-packages/pymongo/uri_parser.py", line 357, in parse_uri
    "begin with '%s' or '%s'" % (SCHEME, SRV_SCHEME))
pymongo.errors.InvalidURI: Invalid URI scheme: URI must begin with 'mongodb://' or 'mongodb+srv://'
unable to load app 0 (mountpoint='') (callable not found or import error)
```

And my PyMongo settings are:
```
u'MONGO_USERNAME': None, 
u'MONGO_PASSWORD': None, 
'MONGO_PORT': 27017, 
'MONGO_DBNAME': u'alignak-backend', 
u'HOST': u'', 
u'PORT': 5000, 
'MONGO_HOST': u'mongo1', 
'MONGO_QUERY_BLACKLIST': ['$where'], 
u'MONGO_URI': u'mongodb://mongoquery1:27017,mongoquery2:27017', 
```

**Comment by rmarren1** [comment]
Thank you! Woke up one morning and my builds were breaking.
Fixed by reverting to `Flask-PyMongo==0.5.2`

**Comment by nicolaiarocci** [comment]
Eve v0.7.10 is out with a pin to Flask-PyMongo dependency.

Thanks!
